.changeset/metal-bottles-lay.md

```diff
@@ -0,0 +1,52 @@
+---
+'@backstage/create-app': patch
+---
+
+Replaced EntitySystemDiagramCard with EntityCatalogGraphCard
+
+To make this change to an existing app:
+
+Add `@backstage/catalog-graph-plugin` in `packages/app`
```

Comment:
```suggestion
Add `@backstage/catalog-graph-plugin` as a `dependency` in `packages/app/package.json`
```

Best be super explicit here to avoid confusion :grin: